libpod/networking_linux.go

```diff
@@ -108,7 +108,11 @@ func (r *Runtime) createNetNS(ctr *Container) (n ns.NetNS, q []*cnitypes.Result,
 	logrus.Debugf("Made network namespace at %s for container %s", ctrNS.Path(), ctr.ID())
 
 	networkStatus := []*cnitypes.Result{}
-	if !rootless.IsRootless() {
+	if rootless.IsRootless() {
```

Comment:
Can we do this check in `options.go` instead, in `WithNetNS()`? Catch it as it's added to the container, instead of when we start?

Reply:
sure, moved and pushed a new version

Reply:
sure, moved and pushed a new version